src/zookeeper/deploy/zookeeper.yaml.template

```diff
@@ -46,6 +46,13 @@ spec:
             - /jobstatus/jobok
           initialDelaySeconds: 5
           periodSeconds: 3
+        env:
+        - name: JAVA_OPTS
+          value: "-server -Xmx512m"
```

Comment:
zookeep container usually consumes 600+M, I suggest raise JVM to 1G and set k8s limits to 1.5G

Reply:
Good, currently it's an initial value, will adjust it soon.